libcaf_core/caf/detail/parser/read_ini.hpp

```diff
@@ -183,16 +184,17 @@ void read_ini_uri(State& ps, Consumer&& consumer) {
   // clang-format on
 }
 
-template <class State, class Consumer>
-void read_ini_value(State& ps, Consumer&& consumer) {
+template <class State, class Consumer, class InsideList>
+void read_ini_value(State& ps, Consumer&& consumer, InsideList inside_list) {
```

Comment:
The type `InsideList` is passed to `read_number_or_timespan` as the type for `EnableRange`, does the tag have a different meaning in both cases?

Reply:
That' correct. We enable the range syntax when inside a list. 🙂